### DISCUSSION TITLE
napi@2.6.0

### DISCUSSION THREAD

**Discussion by Brooooooklyn** [napi-rs/napi-rs#1226] [General]
## What's Changed
* chore(napi): replace lazy_static with once_cell by @jplatte in https://github.com/napi-rs/napi-rs/pull/1213
* feat(napi) Implement `Either3` to `Either26`. by @Hywan in https://github.com/napi-rs/napi-rs/pull/1200
* fix(napi): validate fn for Option<T> by @Brooooooklyn in https://github.com/napi-rs/napi-rs/pull/1223
* feat(napi): clone reference for TypedArray/Buffer by @Brooooooklyn in https://github.com/napi-rs/napi-rs/pull/1224

## New Contributors
* @jplatte made their first contribution in https://github.com/napi-rs/napi-rs/pull/1213

**Full Changelog**: https://github.com/napi-rs/napi-rs/compare/@napi-rs/cli@2.10.0...@napi-rs/cli@2.10.1

<hr /><em>This discussion was created from the release <a href='https://github.com/napi-rs/napi-rs/releases/tag/napi%402.6.0'>napi@2.6.0</a>.</em>

**Comment by Hywan** [comment]
Thanks for the release!
